build-system/server.js

```diff
@@ -561,6 +561,22 @@ app.get('/dist/rtv/99*/*.js', function(req, res, next) {
   }).catch(next);
 });
 
+app.get('/dist/rtv/*/v0/*.js', function(req, res, next) {
+  var mode = getPathMode(req.headers.referer);
+  if (process.argv.includes('test', 2)) {
```

Comment:
That's not a good place. `getPathMode` calculate mode based on the html file path. While here for testing it doesn't apply.